src/components/AllCearButton.js

```diff
@@ -0,0 +1,17 @@
+import React, { Component } from 'react';
```

Comment:
react 18 에서는 `import React from 'react'` 생략이 가능합니다!

```suggestion
import { Component } from 'react';
```